xds/internal/xdsclient/authority.go

```diff
@@ -35,6 +35,7 @@ type watchState int
 
 const (
 	watchStateStarted watchState = iota
+	watchStateRespRequested
```

Comment:
s/watchStateRespRequested/watchStateRequested/ (response requested does not make much sense).

Also, might be good time to add some comments to these constants. Something like:

```golang
const (
	watchStateStarted watchState = iota	// Watch started, request not yet set.
	watchStateRespRequested			// Request sent for resource being watched.
	watchStateRespReceived			// Response received for resource being watched.
	watchStateTimeout				// Watch timer expired, no response.
	watchStateCanceled				// Watch cancelled.
)
```